src/org/openstreetmap/josm/gui/tagging/presets/TaggingPreset.java

```diff
@@ -605,6 +601,25 @@ public int showDialog(Collection<OsmPrimitive> sel, boolean showNewRelation) {
 
         int answer = 1;
         boolean canCreateRelation = types == null || types.contains(TaggingPresetType.RELATION);
+        if(regions!=null && !sel.isEmpty()){
```

Comment:
```suggestion
        if (Utils.isEmpty(this.regions)) {
```
We have a specific utility method for checking if a collection is `null` or `isEmpty`.